R/load_forecasts.R

```diff
@@ -34,7 +36,8 @@ load_forecasts <- function (
   source = "zoltar", 
   hub_repo_path,
   as_of = NULL,
-  verbose = TRUE) {
+  verbose = TRUE, 
+  hub = c("US", "ECDC")) {
```

Comment:
standardizing order of arguments across functions
```suggestion
  hub = c("US", "ECDC"),
  verbose = TRUE) {
```